src/java/com/phenix/pct/BackgroundWorker.java

```diff
@@ -61,7 +62,7 @@ public final void initialize(Socket socket) throws IOException {
         this.reader = new BufferedReader(new InputStreamReader(this.socket.getInputStream(), parent.getCharset()));
         this.writer = new BufferedWriter(new OutputStreamWriter(this.socket.getOutputStream(), parent.getCharset()));
 
-        // FIXME Should be HELLO or something like that...
+        // TODO Should be HELLO or something like that...
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1135)